fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/QuerySegmentAggregator.java

```diff
@@ -353,10 +352,10 @@ protected String buildWhereClause(String overrideType) {
 
                     whereClauseSegment = querySegment.getQueryString();
                     if (Modifier.MISSING.equals(param.getModifier())) {
-                        whereClause.append(AND).append(whereClauseSegment);
+                        whereClause.append(whereClauseSegment);
```

Comment:
should .replaceAll(PARAMETER_TABLE_ALIAS + ".", "") be called here?  or it's already fixed?

Reply:
px (PARAMETER_TABLE_ALIAS) is not using in processingMissingParm, so it's not needed.